### PR TITLE
Swagger changes for Stable version(2020-09-01) for Microsoft.DataBoxEdge

### PR DESCRIPTION
**Pull request by pvsraviteja**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by yungezz** [comment]
wait for ARM feedback. hi @msyyc, could you pls check impact to Python SDK? thanks

**Comment by pvsraviteja** [comment]
> wait for ARM feedback. hi @msyyc, could you pls check impact to Python SDK? thanks

We don't have any end customers using python SDK. So, we should be ok with this.

**Comment by msyyc** [comment]
> > wait for ARM feedback. hi @msyyc
> >             
> >             Yuchao Yan
> >             FTE, could you pls check impact to Python SDK? thanks
> 
> We don't have any end customers using python SDK. So, we should be ok with this.

It's ok for python

**Comment by mentat9** [comment]
In future, please commit unchanged files first so diffs against base are easy.

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json [hunk 1]
```diff
@@ -0,0 +1,8511 @@
...
+    },
+    "GenerateCertResponse": {
+      "description": "Used in activation key generation flow.",
+      "type": "object",
+      "properties": {
+        "publicKey": {
+          "description": "Gets or sets base64 encoded certificate raw data,\r\nthis is the public part needed to be uploaded to cert vault",
+          "type": "string"
+        },
+        "privateKey": {
```
Comment (j5lim) [inline specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:5280]: Please annotate with x-ms-secret.

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json [hunk 2]
```diff
@@ -0,0 +1,8511 @@
...
+            "description": "The metrics configuration is already deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
```
Comment (j5lim) [inline specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:2032]: Doesn't the API return 'Location' header to track async DELETE?
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#delete-resource-asynchronously

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json [hunk 3]
```diff
@@ -0,0 +1,8511 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-09-01",
```
Comment (j5lim) [inline specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:4]: Looks like it makes the same changes as in https://github.com/Azure/azure-rest-api-specs/pull/12350. Please apply the same comments left for the preview version.

**Review by j5lim — CHANGES_REQUESTED** [review]

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json (hunk 1 above)
Comment (pvsraviteja) [reply specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:5284]: Changed

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json (hunk 2 above)
Comment (pvsraviteja) [reply specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:2035]: Yes it returns location header to track the requests

File: specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json (hunk 3 above)
Comment (pvsraviteja) [reply specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json:4]: Changes are pushed.

**Review by j5lim — APPROVED** [review]
Signed from ARM.

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @pvsraviteja is service change deployed to Prod already? If yes, will merge the PR.

**Comment by yungezz** [comment]
hi @pvsraviteja could you pls rebase? there's conflict

**Comment by pvsraviteja** [comment]
> hi @pvsraviteja could you pls rebase? there's conflict

Sure let me do that

**Comment by pvsraviteja** [comment]
> hi @pvsraviteja is service change deployed to Prod already? If yes, will merge the PR.

Yes they are deployed to prod and ready

**Pull request merged**